addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/cli/Parser.java

```diff
@@ -35,8 +35,7 @@
     private static final Logger logger = LoggerFactory.getLogger(Parser.class);
 
     private static final Pattern pattern = Pattern.compile("^\\s+([a-z\\s._]+)[:=]\\s*<?\"?([^>\"]+)\"?>?$");
-    private static final Pattern volumePattern = Pattern.compile(
-            "^(0|front-left):( *[0-9]+ \\/)? *([0-9]+)% *\\/? *([0-9\\-, dB]+)?(1|front-right):( *[0-9]+ \\/)? *([0-9]+)% *\\/? *([0-9\\-, dB]+)?$");
+    private static final Pattern volumePattern = Pattern.compile("^([\\w\\-]+):( *[\\d]+ \\/)? *([\\d]+)% *\\/? *([\\d\\-., dB]+)?$");
```

Comment:
CLI vs. TCP:
Well its just easier to use. At the time I wrote the first version of this binding the CLI interface was much easier to use and did everything I needed (and that doesn't seem to have changed yet). I still don't know or understand if and how I can achieve the same things with the native interface.

I always had the little hope that some time someone with more experiance in pulseaudio, would re-write this binding to "do it right" but that never happened. So if you are the one, feel free to start ;-)

Regex change:
Ok you have convinced me, I havn't thought about the multichannel stuff, so your solution should work in more environments that the old one.